src/it/test-dependencies-transitive-optional/pom.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>4.40</version>
+        <relativePath/>
+    </parent>
+    <groupId>org.jenkins-ci.tools.hpi.its</groupId>
+    <artifactId>test-dependencies-transitive-optional</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>hpi</packaging>
+    <properties>
+        <!-- change it back to @jenkins.version.test@ when upgrade made in projec pom -->
+        <jenkins.version>2.346.1</jenkins.version>
+        <hpi-plugin.version>@project.version@</hpi-plugin.version>
+        <useUpperBounds>true</useUpperBounds>
+        <test>SampleTest</test>
+        <maven.test.redirectTestOutputToFile>false</maven.test.redirectTestOutputToFile>
+        <surefire.rerunFailingTestsCount>0</surefire.rerunFailingTestsCount>
+        <jenkins.war.directory>${project.build.directory}/war</jenkins.war.directory>
+        <jenkins.war.name>jenkins-${jenkins.version}.war</jenkins.war.name>
+        <overrideWar>${jenkins.war.directory}/${jenkins.war.name}</overrideWar>
+    </properties>
+    <repositories>
+        <repository>
+            <id>repo.jenkins-ci.org</id>
+            <url>https://repo.jenkins-ci.org/public/</url>
+        </repository>
+    </repositories>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>repo.jenkins-ci.org</id>
+            <url>https://repo.jenkins-ci.org/public/</url>
+        </pluginRepository>
+    </pluginRepositories>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>copy-jenkins-war</id>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>copy</goal>
```

Comment:
This does not look anything like a real plugin POM so I am struggling to follow what bug this IT purports to reproduce.